packages/react-components/react-portal/src/virtualParent/getParent.ts

```diff
@@ -3,12 +3,15 @@ import { isVirtualElement } from './isVirtualElement';
 type GetParentOptions = {
   /**
    * Indicates if getParent() should ignore a virtual parent.
+   * @internal
```

Comment:
Please re-export utils instead of just keeping them.

Reply:
done